ui/admin-portal/src/app/components/candidates/show/show-candidates.component.ts

```diff
@@ -506,7 +506,6 @@ export class ShowCandidatesComponent implements OnInit, OnChanges, OnDestroy {
         request.pageSize = this.pageSize;
         request.sortFields = [this.sortField];
         request.sortDirection = this.sortDirection;
-        request.dtoType = DtoType.PREVIEW;
```

Comment:
This means that we will always do full fetches of candidate data when displaying lists of candidates - removing all performance optimizations. Was that intentional?

Reply:
Yes, this was intentionally reverted to a full fetch for this PR, but it’s not my intention to leave it that way for the release. I should have been clear about that in the PR comments. 

The thought was to fix the 2 bugs as quickly as possible in staging; and take a more careful review of what we need to and don't need to send for show-candidote previews, including selectively sending only the minimum required opportunity fields, in a separate issue/PR. 

However, given this PR is not yet merged, at this stage it may be better to do it all in this PR after all. Will take stock in the morning.

Reply:
I've pushed changes to revert those methods under test mentioned above to private. Additionally, I've opened a new ticket to address the show-candidates preview DTO fields (#1321). I think we can now merge this PR's changes to staging and pick up show-candidates previews as #1321.